FreeRTOS/FreeRTOS-Kernel
# Consequent TickType_t  stack types

Consequent usage of `TickType_t` and `configSTACK_DEPTH_TYPE`.
-----------
Currently `uint32_t` is used in several places, where more accurately`TickType_t` and `configSTACK_DEPTH_TYPE` should be used. The correction is relevant in ports that have either 16 bit or 64 bit tick timers or 16 bit stack depths.

Test Steps
-----------
Improved definition.
Optimisation where 16 bit variables and parameters can be handled rather than 32 bit variables and parameters.

-----------
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
